### DISCUSSION TITLE
Help - How can i export customer, project, activities and custom fields from one system to another

### DISCUSSION THREAD

**Discussion by rarvin** [kimai/kimai#3197] [Questions & Answers]
Dear Kevin,
I am in need of setting up a new system, but I do not need any of the data from the old system except customer, project, activities and all of the custom fields (I have a plugin and have configured fileds) from one system to another. No users and user data or timesheet data is needed.

Please help.

**Comment by kevinpapst** [comment]
You can install the new system, then create a data dump (from the old system ) only for the tables that you need.

```
kimai2_(customers/projects/activities)
kimai2_(customers/projects/activities)_rates
kimai2_(customers/projects/activities)_meta
kimai2_(customers/projects)_comments
kimai2_meta_field_rules
```
Import this data into the new system. Shouldn't be a problem.

Remember: additional prod system requires new plugin licenses^^
